### ISSUE TITLE
Changing default resolution size changes the size of view in a new tab

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#6955]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Changing default resolution size changes the size of view in a new tab

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version (revision SHA):
Brave	0.13.1
rev	a44d043

- Steps to reproduce:
    1. Do a clean install of 0.13.0/0.13.1
    2. Change default zoom level to anything less than 100% say 90%
    3. Open a new tab, webview is limited
    4. Open a site in the new tab, page is only loaded in the limited webview
    5. Maximize or restore, webview is shown fully
    
- Actual result:
Webview is reduced and the page only loads in that limited webview

- Expected result:
Page should be opened in full tab view

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes on 0.13.0. Verified on 0.12.15 doesn't happen

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
![image](https://cloud.githubusercontent.com/assets/17010094/22482957/21b7465e-e7f3-11e6-8272-8e3a76398e6e.png)
![image](https://cloud.githubusercontent.com/assets/17010094/22482985/40801afc-e7f3-11e6-8dbc-e0d0fd11e933.png)

- Any related issues:
cc: @bsclifton

**Comment by luixxiul** [comment]
I confirmed it too

**Comment by Liunkae** [comment]
Still exists with Brave 0.18.x.

**Comment by Gnat008** [comment]
This continues to be an issue on Linux as of 0.18.16 (rev 8003c66).

In my use case, the default zoom has to be 125% to properly fit on the monitor. When a new tab is opened, everything is moved to the right, with part of the page and the scrollbar entirely off the screen, as shown: https://imgur.com/Cm6p01W

Needless to say, this makes it a little annoying to use. Some focus in this area would be wonderful. :)

**Comment by Gnat008** [comment]
Here's a puzzling update... I can no longer reproduce this issue. No updates have been installed, the Brave version and revision is the same. But new tabs open and display correctly at the proper scaling and location.

The only change is I had logged out last night, and logged in this morning.

**Comment by Gnat008** [comment]
Testing on a fresh VM running ElementaryOS Loki 0.4.1 confirmed my results above.

On the first login, installed Brave. Changed the zoom setting to 125%, and opened a new tab and pointed it to npr.org. Everything was pushed to the right, with part of the site off-screen. Then I logged out of Elementary, and back in, and reopened Brave, with the previous tabs closed, zoom still at 125%. Opened a new tab to npr.org, and everything displayed normally.

One thing I did not test is if the bug reappeared after the relog by changing to different zoom levels.
